package.json

```diff
@@ -3,9 +3,10 @@
   "version": "1.0.2",
   "description": "Sophisticated and functionally-minded async with advanced features: coroutines, promises, ES2015 iterables, fantasy-land",
   "main": "dist/creed.js",
-  "jsnext:main": "src/main.js",
+  "jsnext:main": "dist/creed.es.js",
```

Comment:
Isn't the point of `jsnext:main` to allow better tree-shaking or even native ES6 module imports?